tests/test_server.py

```diff
@@ -72,25 +72,30 @@ def test_status_endpoint_with_all_files_analyzed(server, snapshot):
     assert data == snapshot
 
 
+import pytest
+
 @pytest.mark.usefixtures("repo_with_more_files")
-def test_status_endpoint_with_some_files_not_analyzed(server):
```

Comment:
I guess in the end it would be nice to still keep `test_status_endpoint_with_some_files_not_analyzed` while also including the test that you wrote